src/tests/testSetup.ts

```diff
@@ -0,0 +1,60 @@
+import { runMutations } from "automutate";
+import fs from "node:fs";
+import path from "node:path";
+import ts from "typescript";
+
+import { fillOutRawOptions } from "../options/fillOutRawOptions.js";
+import { RawTypeStatOptions } from "../options/types.js";
+import { createTypeStatProvider } from "../runtime/createTypeStatProvider.js";
+
+export const runMutationForTest = async (
```

Comment:
[Naming] Since this function really is running the entire mutation test, I'd say...

```suggestion
export const runMutationTest = async (
```

WDYT?